diff --git a/WORKSPACE b/WORKSPACE
index e4d29bb3..86dd27e5 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -13,10 +13,18 @@ workspace(name = "tf_serving")
 # 3. Request the new archive to be mirrored on mirror.bazel.build for more
 #    reliable downloads.
 load("//tensorflow_serving:repo.bzl", "tensorflow_http_archive")
-tensorflow_http_archive(
-    name = "org_tensorflow",
-    sha256 = "21d919ad6d96fcc0477c8d4f7b1f7e4295aaec2986e035551ed263c2b1cd52ee",
-    git_commit = "3f878cff5b698b82eea85db2b60d65a2e320850e",
+#tensorflow_http_archive(
+#    name = "org_tensorflow",
+#    sha256 = "21d919ad6d96fcc0477c8d4f7b1f7e4295aaec2986e035551ed263c2b1cd52ee",
+#    git_commit = "3f878cff5b698b82eea85db2b60d65a2e320850e",
+#)
+local_repository(
+      name = "org_tensorflow",
+      path = "$SOURCE_ROOT/tensorflow",
+)
+local_repository(
+      name = "boringssl",
+      path = "$SOURCE_ROOT/boringssl",
 )

 # Import all of TensorFlow Serving's external dependencies.
